### PR TITLE
true: add page

### PR DESCRIPTION
**Pull request by pxgamer**
Part of #2213

----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/linux/true.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# true
+
+> Does nothing, but return a successful exit status code.
```
Comment (sbrl) [inline pages/linux/true.md]: Perhaps ...`but returns a successful exit status code of 0`... would read better?

File: pages/linux/true.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# true
+
+> Does nothing, but returns a successful exit status code of 0.
```
Comment (agnivade) [inline pages/linux/true.md]: Let us remove "Does nothing, but".

Also, can we add a sub-heading mentioning - "Use this with the || operator to make a command always exit with 0" ?

Comment (sbrl) [reply pages/linux/true.md]: Can do!

**Comment by melgu** [comment]
The `true` and `false` commands also exist on macOS. Wouldn't it be better to place those in `common/` instead of `linux/`?

**Comment by agnivade** [comment]
Do they exist on WSL ?

**Comment by pxgamer** [comment]
@agnivade, pretty sure they do exist on WSL. So maybe I'll move them to `common`. I'll check in about 25 minutes when I'm at work. :+1:

**Comment by pxgamer** [comment]
I can confirm that it is available on WSL. 👍

**Pull request merged**